src/components/BreedsList.vue

```diff
@@ -93,30 +87,39 @@ export default {
     },
     getSearchResults(input) {
       this.isLoading = true;
-      setTimeout(() => {
-        if (this.isLoading) {
-          this.isTimeOut = true;
-          this.isLoading = false;
-        }
-      }, 1000);
+      this.message = "Data is loading...";
       this.searchQuery = input;
 
+      if (this.searchQuery.length === 0) {
+        this.isLoading = false;
+      }
+
       searchBreed(this.searchQuery)
         .then((results) => {
-          this.isTimeOut = false;
           this.isLoading = false;
-          this.searchList = results;
           if (results.length > 0) {
+            results.map((breed) => {
+              breed.isActive = false;
+            });
+            this.searchList = results;
```

Comment:
lahko tko kot sem napisal zgorej.. 

imaš podvojeno `this.searchList = results;`